tests/python_tests/samples/test_text2speech.py

```diff
@@ -34,6 +35,10 @@ def teardown_class(self):
     @pytest.mark.precommit
     @pytest.mark.parametrize("convert_model", ["tiny-random-SpeechT5ForTextToSpeech"], indirect=True)
     @pytest.mark.parametrize("input_prompt", ["Hello everyone"])
+    @pytest.mark.xfail(
```

Comment:
If it's not ci issue, then speech generation wont work with deps update. Should this be a blocker?

Reply:
Then it's blocker for the moment

Reply:
Sure, agree